backoffice/src/Controllers/EmailController.cs

```diff
@@ -17,9 +17,13 @@ public EmailController(ILogger<EmailController> logger, IEmailService service)
         _service = service;
     }
 
-    public IActionResult Index()
+    public async Task<IActionResult> Index()
     {
-        return View();
+        var emails = await _service.GetEmails();
+
+        var model = emails.Select(e => new EmailView(e)).ToList();

```

Comment:
You'll find down the line you'll want to add additional data to this view. 

Pass in something like an EmailListView model, that has a list of EmailViews on them, rather than just the list of EmailViews

Reply:
applied